diff --git a/src/rpc/rawtransaction.cpp b/src/rpc/rawtransaction.cpp
index 461e8025d..ff8deb01d 100644
--- a/src/rpc/rawtransaction.cpp
+++ b/src/rpc/rawtransaction.cpp
@@ -809,7 +809,8 @@ static UniValue sendrawtransaction(const JSONRPCRequest& request)
     CFeeRate max_raw_tx_fee_rate = DEFAULT_MAX_RAW_TX_FEE_RATE;
     // TODO: temporary migration code for old clients. Remove in v0.20
     if (request.params[1].isBool()) {
-        throw JSONRPCError(RPC_INVALID_PARAMETER, "Second argument must be numeric (maxfeerate) and no longer supports a boolean. To allow a transaction with high fees, set maxfeerate to 0.");
+// work around https://github.com/btcsuite/btcwallet/issues/611
+//        throw JSONRPCError(RPC_INVALID_PARAMETER, "Second argument must be numeric (maxfeerate) and no longer supports a boolean. To allow a transaction with high fees, set maxfeerate to 0.");
     } else if (!request.params[1].isNull()) {
         max_raw_tx_fee_rate = CFeeRate(AmountFromValue(request.params[1]));
     }
