diff --git a/cmd/btcctl/config.go b/cmd/btcctl/config.go
index 3db735c..0710f20 100644
--- a/cmd/btcctl/config.go
+++ b/cmd/btcctl/config.go
@@ -133,9 +133,7 @@ func normalizeAddress(addr string, chain *chaincfg.Params, useWallet bool) (stri
 			}
 		case &chaincfg.RegressionNetParams:
 			if useWallet {
-				// TODO: add port once regtest is supported in btcwallet
-				paramErr := fmt.Errorf("cannot use -wallet with -regtest, btcwallet not yet compatible with regtest")
-				return "", paramErr
+				defaultPort = "18443"
 			} else {
 				defaultPort = "18334"
 			}
