diff --git a/tools/docker-entrypoint.sh b/tools/docker-entrypoint.sh
index c5bccf4..5d91839 100755
--- a/tools/docker-entrypoint.sh
+++ b/tools/docker-entrypoint.sh
@@ -2,7 +2,7 @@
 
 : "${EXPOSE_TCP:=false}"
 
-networkdatadir="${LIGHTNINGD_DATA}/${LIGHTNINGD_NETWORK}"
+networkdatadir=${LIGHTNINGD_RPC_DIR_SIMVERSE:-"${LIGHTNINGD_DATA}/${LIGHTNINGD_NETWORK}"}
 
 if [ "$EXPOSE_TCP" == "true" ]; then
     set -m
