diff --git a/Makefile b/Makefile
index 7e23e8a2..a1c53b9d 100644
--- a/Makefile
+++ b/Makefile
@@ -4,7 +4,7 @@
 VERSION=$(shell git describe --always --dirty=-modded --abbrev=7 2>/dev/null || pwd | sed -n 's,.*/lightning-\([0-9.rc]*\)$$,\1,p')
 
 ifeq ($(VERSION),)
-$(error "ERROR: git is required for generating version information")
+VERSION="unknown"
 endif
 
 DISTRO=$(shell lsb_release -is 2>/dev/null || echo unknown)-$(shell lsb_release -rs 2>/dev/null || echo unknown)
